src/main/java/mike/cli/CommandLine.java

```diff
@@ -94,20 +94,36 @@ public static void commandInterface() {
 				// Call create class, attribute, or relationship based on length and user input
 				case "create":
 					if (commands[1].equals("class")) {
-						if(commands.length != 3 || !userClasses.createClass(commands[2])) {
+						if(commands.length != 3) {
 							System.out.println(errorMessage + commandUsage[2] + "\n");
+							break;
+						}
+						if(!userClasses.createClass(commands[2])){
+							System.out.println("\nCreate class failed. Make sure the class name doesn't already exist.\n");
 						}
 					} else if ( commands[1].equals("field")) {
-						if(commands.length != 4 || !userClasses.createField(commands[2], commands[3])) {
+						if(commands.length != 4) {
 							System.out.println(errorMessage + commandUsage[3] + "\n");
+							break;
+						}
+						if(!userClasses.createField(commands[2], commands[3])){
+							System.out.println("\nCreate field failed. Make sure the field doesn't already exist and the class name does exists.\n");
```

Comment:
Just a typo - delete the "s" at the end of exists

"Make sure the method doesn't already exist, and the class name does exist**s**"